pubsubs/registry.py

```diff
@@ -10,14 +10,14 @@ def __init__(self):
         self._registry = {}
 
     def new(self, *, name, backend, **kwargs):
-        """ Register concrete instance."""
+        """ Register new instantiated concrete instance."""
         return self.register(
             MessageQueue.for_backend(backend)(name, registry=self, **kwargs).connect(),
             name=name,
         )
 
     def register(self, message_queue, name):
-        """ Register concrete message queue instance under a name."""
+        """ Assign concrete concrete instance to a name."""
         name = message_queue.name if not name else name
```

Comment:
In `def register(self, message_queue, name):` do you want a default `name=None`? else to trigger the first part of
```
name = message_queue.name if not name else name
```
would you have to do `register(queue="kafka", name=None)`

Reply:
I'm going to drop this check